src/base-input.js

```diff
@@ -709,6 +709,14 @@ export default class BaseInput extends LitElement {
     }
 
     this.getErrorMessage();
+
+    this.dispatchEvent(new CustomEvent('auroInput-validated', {
+      bubbles: true,
+      composed: true,
+      detail: {
+        validity: this.validity
+      }
+    }));
```

Comment:
I noted in the issue that this was removed in a previous PR, but what I find interesting is that this code is included in the larger `validate()` method, but not nuanced enough to really focus on this dispatched event.

Reply:
I created an issue for this code to be tested better. 

- https://github.com/AlaskaAirlines/auro-input/issues/235